### ISSUE TITLE
Acorn crashes when you have an invalid profile image url.

### ISSUE THREAD

**Issue by TechSupport26** [lightningrodlabs/acorn#230]
This only happens in this one project but it has 100 odd cards.

When I access it acorn crashes as soon as I move my mouse or perform any user input including Ctrl+Shift (It crashes before I can press r) but not Ctrl+R. This makes the project inaccessible.
This is affecting other projects slightly as when I open a different project after leaving the first one the new one crashes. It is fixed after a refresh.

Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The HTMLImageElement provided is in the 'broken' state.

Error: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The HTMLImageElement provided is in the 'broken' state.
    at file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94566:17
    at draw (file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94095:5)
    at file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94534:9
    at draw (file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94095:5)
    at drawAvatar (file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94484:103)
    at file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94611:17
    at Array.forEach (<anonymous>)
    at file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94602:21
    at draw (file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94095:5)
    at drawAssignees (file:///C:/Users/zephd/AppData/Local/Programs/acorn/resources/app.asar.unpacked/web/app.js:94600:9)

![image](https://user-images.githubusercontent.com/65631811/209587077-1de414ca-aff8-40b1-9a9c-76bc0fb3e777.png)
![image](https://user-images.githubusercontent.com/65631811/209587085-0264a4d6-e952-46a3-b8e0-23e5c54385c4.png)

You will see in the image the error https://ibb.co/jZ9pXv5 that is the image that I was trying to use for my profile however it was not loading so it caused the Acorn to crash. I think that because the image was not able to load it stopped Acorn from recalculating computedOutcomes!
After I removed my profile image Acorn loaded the project with no issues.

I want to try this with a second user to see if it will affect someone over the network. I will let you know what that results in.
So that is my story and bug report.

TL;DR Acorn crashes if a project is too large and you use a bad user profile image url.

**Comment by Connoropolous** [comment]
you are right. I've seen this issue. It's a must fix asap

**Comment by TechSupport26** [comment]
I just wanted to update this a little.
In version v4.0.0 if you have an invalid profile picture URL, it will cause other members that have that same project open to crash until the faulty profile picture is changed and updated for the other users. (It also crashes Acorn for the user of the bad profile picture)
This happens regardless of the size of the project.
